sysbuild/secureboot.cmake

```diff
@@ -7,34 +7,62 @@ get_property(PM_DOMAINS GLOBAL PROPERTY PM_DOMAINS)
 if(SB_CONFIG_SECURE_BOOT)
   if(SB_CONFIG_SECURE_BOOT_NETCORE)
     set(secure_boot_source_dir ${ZEPHYR_NRF_MODULE_DIR}/samples/nrf5340/netboot)
-  else()
+
+    ExternalZephyrProject_Add(
+      APPLICATION b0n
+      SOURCE_DIR ${secure_boot_source_dir}
+      BOARD ${SB_CONFIG_SECURE_BOOT_NETWORK_BOARD}
+      BUILD_ONLY true
+    )
+    set_target_properties(${SB_CONFIG_SECURE_BOOT_IMAGE_NAME} PROPERTIES
+      IMAGE_CONF_SCRIPT ${CMAKE_CURRENT_LIST_DIR}/image_configurations/b0_image_default.cmake
+    )
+
+    # Can we do this in a more clever way ?
+    # Should adding of hci_rpmasg be a downstream feature, or could we perhaps
+    # overload upstream image adding.
```

Comment:
Should be a JIRA ticket and not a comment here.

Reply:
Removed